src/components/GlassWall/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box } from '@chakra-ui/react';
+import Image from 'next/image';
+
+export const glassWallTestIds = {
+  base: 'glass-wall',
+};
+
+interface GlassWallProps {
+  src: string;
+  alt?: string;
+}
+
+const GlassWall = ({ src, alt }: GlassWallProps) => {
+  return (
+    <Box
+      pos="absolute"
+      w="100vw"
+      h="100vh"
+      minH="full"
+      top="0"
+      zIndex="hide"
+      data-test-id={glassWallTestIds.base}
+    >
+      <Image src={src} alt={alt} layout="fill" objectFit="cover" />
+      <Box
+        pos="absolute"
+        w="100%"
+        h="100%"
+        bgGradient="linear(to-b, blackAlpha.350, black)"
+        backdropFilter="blur(100px)"
```

Comment:
@carryall  As Thumb is using `backdrop-filter`, so to display it on Firefox, you should enable it from the config:
![backdrop-filter_-_CSS__Cascading_Style_Sheets___MDN](https://user-images.githubusercontent.com/7344405/172990694-703fa032-c9d7-42c2-8e52-77ffef15ed43.png)

https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter

Reply:
@hoangmirs I overrided with that configuration but it didn't work 🥲
I think I'll fix with the other way with this [blur package](https://github.com/javierbyte/react-blur) later

@carryall I draft a ticket here
![Screen Shot 2565-06-10 at 12 30 06](https://user-images.githubusercontent.com/749672/172996839-0fc228ad-bf2b-4180-a59c-c8cadd95208c.png)